nss-day-cohort-38/welcome-to-nashville-ghost-ducks
# cn-restaurants

Directions to test:

1. git fetch --all
2. git checkout cn-restaurants
3. use code . and hs -o to check
4. Search bar should take in a value for a type of cuisine (Thai, Italian, Burgers, etc.) and render results to the dom. It should not be case sensitive.
